expected-consensus.md

```diff
@@ -352,7 +433,7 @@ potentially long chain scans would be required to compute a given block's weight
 
 ### Selecting between TipSets with equal weight
 
-When selecting between TipSets of equal weight, a miner chooses the one with the smallest min ticket (by bytewise comparison).
+When selecting between TipSets of equal weight, a miner chooses the one with the smallest min ticket.
```

Comment:
My reading of the above discussion is that tickets are comparable against tickets but I don't know how to compare ticket arrays with each other.  Comparing ticket arrays is the thing we really want for 1) tie breaking, 2) consistent message application order.  It is the thing we should include in the spec.

Right now I'm imagining we just take the first ticket of the array and compare.

Reply:
Reading above I'm actually guessing its the final ticket, not the first.  But let's make sure to make that clear.

Reply:
Understood. Yes! Thanks